src/components/ImageView/ImageView.js

```diff
@@ -741,6 +748,10 @@ export default observer(
         containerStyle["height"] = item.height;
       }
 
+      if(!this.props.store.settings.enableSmoothing && item.zoomScale > 1.4){
```

Comment:
why 1.4? if we do this only for zoomed in images it should work right after 1

Reply:
@hlomzik I was testing with 1.4 and I forgot to change to 1, I'll change! Thanks!!